script/ci-build

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -v
+
+export DIR=`pwd`
+
+if [ "x$RUBY_SASS" = "xyes" ]; then
```

Comment:
This ancient trick is necessary when doing $var = yes to avoid a syntax error, "${VAR}" = "yes" should do